app/templates/base/home.html

```diff
@@ -12,8 +12,10 @@
       <li class="tab-superuser">Superuser</li>
     {% endif %}
     <li class="tab-analytical-tools">Analytical tools</li>
-    {% if current_user.is_superuser %}
+    {% if env.ENV == 'dev' or current_user.is_superuser %}
```

Comment:
No need to change anything, but feature flags should preferably be specific and obviously named - eg: `env.ENABLE_WAREHOUSE_DATA_TAB`. Hanging a bunch of features off `ENV=dev` could potentially be a problem if you wanted to enable one but not another.